cmd/podman/generate/systemd.go

```diff
@@ -33,6 +33,11 @@ const (
 )
 
 var (
+	deprecation = `
+DEPRECATED command:
+It is recommended to use Quadlets for running containers and pods under systemd.
```

Comment:
Please point to the Quadlet man page.  Some users may not be aware of quadlet at all.